type-definitions/index.d.ts

```diff
@@ -20,5 +20,5 @@ declare interface DisposeFn {
 }
 
 export default function create<A>(f: (add: (a: A) => any,
-                                      end: (x: any) => any,
+                                      end: (x?: any) => any,
```

Comment:
Needs to be of type `A`, tho.  These are basically "functionified" versions of the [Sink methods](https://github.com/cujojs/most/blob/master/type-definitions/most.d.ts#L24), so I think we can make the return type `void`:

```js
export default function create<A>(f: (add: (a: A) => void,
                                      end: (a?: A) => void,
                                      error: (e: Error) => void) => void | DisposeFn ): Stream<A>;
```